frameworks/pytorch/get_started_mnist_train.ipynb

```diff
@@ -0,0 +1,408 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MNIST training with PyTorch\n",
+    "\n",
+    "MNIST is a widely used dataset for handwritten digit classification. It consists of 70,000 labeled 28x28 pixel grayscale images of hand-written digits. The dataset is split into 60,000 training images and 10,000 test images. There are 10 classes (one for each of the 10 digits). This tutorial will show how to train and test an MNIST model on SageMaker using PyTorch. \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "\n",
+    "import sagemaker\n",
+    "from sagemaker.pytorch import PyTorch\n",
+    "from sagemaker import get_execution_role\n",
+    "\n",
+    "\n",
+    "sess = sagemaker.Session()\n",
+    "\n",
+    "role = get_execution_role()\n",
+    "\n",
+    "output_path='s3://' + sess.default_bucket() + '/mnist'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## PyTorch Estimator\n",
+    "\n",
+    "The `PyTorch` class allows you to run your training script on SageMaker\n",
+    "infrastracture in a containerized environment. In this notebook, we\n",
+    "refer to this container as *training container*. \n",
+    "\n",
+    "You need to configure\n",
+    "it with the following parameters to set up the environment:\n",
+    "\n",
+    "- entry_point: A user defined python file to be used by the training container as the \n",
+    "instructions for training. We further discuss this file in the next subsection.\n",
+    "\n",
+    "- role: An IAM role to make AWS service requests\n",
+    "\n",
+    "- instance_type: The type of SageMaker instance to run your training script. \n",
+    "Set it to `local` if you want to run the training job on \n",
+    "the SageMaker instance you are using to run this notebook\n",
```

Comment:
Add comment that local mode is not supported in Studio.

Reply:
A comment is added before calling `.deploy`